atlp-rwanda/atlp-pulse-fn
# #DP-106 Feat: Rate trainees

### PR Description
- #76 
This pull request introduce the changes that allow the coordinator to add ratings to trainee
### Description of tasks that were expected to be completed
- Add ability to rate trainees
- The integration with the frontend is working
### How has this been tested?
- Run ( npm run dev on the front-end and backend )
- Login an organization ( ex: andela) in the browser
- Sign in a coordinator ( ex: coordinator@devpulse.co , password: Andela123)
- navigate to trainee, add trainee in the cohort , refer to this image
- 
<img width="1427" alt="Screen Shot 2022-10-12 at 09 53 28" src="https://user-images.githubusercontent.com/79087014/195283774-c64764cb-7ee0-4095-b6be-7306e81dd577.png">

- Then navigate to Rating ( Select cohort, click on add ratings, rate the trainee and save ) refer here
- 
<img width="1427" alt="Screen Shot 2022-10-12 at 10 56 30" src="https://user-images.githubusercontent.com/79087014/195298312-b36c942b-8248-4c9d-95d5-e10d65b06e9b.png">
- Next test edit functionality by clicking edit icon in the column of actions, edit the ratings, refer here
- 
<img width="1427" alt="Screen Shot 2022-10-12 at 11 06 03" src="https://user-images.githubusercontent.com/79087014/195300492-f6e4a9f0-289c-4e61-8f75-9fae13723c5c.png">

- logout the coordinator,  Login admin (ex: admin@devpulse.co, password: Andela123), 
- check on update ratings from sidebar, refer here
- 
<img width="1427" alt="Screen Shot 2022-10-12 at 11 11 15" src="https://user-images.githubusercontent.com/79087014/195301775-221c29d1-f50c-44ed-8d1f-bafe32a71d57.png">
- approve or reject the updates
- Logout the admin and login trainee to check his/her ratings( i.e login trainee with ratings), check performance refer here
- 
<img width="1427" alt="Screen Shot 2022-10-12 at 11 15 45" src="https://user-images.githubusercontent.com/79087014/195302859-47920def-b566-402d-b76b-5dce529e70a9.png">

- Click on details to view remarks given on the performance, refer here
- 
<img width="1427" alt="Screen Shot 2022-10-12 at 11 17 18" src="https://user-images.githubusercontent.com/79087014/195303170-ca2da660-d085-4518-bcc8-fdc2d1f59698.png">


### Screenshots (If appropriate)
